kerchunk/grib2.py

```diff
@@ -1,18 +1,17 @@
 import base64
 import logging
-import os
-import tempfile
 
 import cfgrib
-import numcodecs.abc
-from numcodecs.compat import ndarray_copy, ensure_contiguous_ndarray
+import eccodes
 import fsspec
 import zarr
 import numpy as np
 
 from kerchunk.utils import class_factory
+from kerchunk.codecs import GRIBCodec
 
 logger = logging.getLogger("grib2-to-zarr")
+fsspec.utils.setup_logging(logger=logger)
```

Comment:
Is this appropriate to do in a library?
I don't think you want to add your own handlers here.

Reply:
Ah sorry, this does seem to be from my own debugging. Generally, people probably do want this logging, but indeed it should be up to the caller to decide.

Reply:
Will you release 0.0.7 with these changes?

Reply:
It's on my list :)

Reply:
Released - thank you!